### PR TITLE
Gofumpt

### PR DESCRIPTION
**Pull request by faddat**
## Description

Gofumpt is useful for cleaning the code, and is included in the gitpod docker container.

Merge second.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/pylons/client/cli/query_list_item_by_owner_test.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func TestListItemByOwner(t *testing.T) {
 				require.NoError(t, err)
 				var resp types.QueryListItemByOwnerResponse
 				require.NoError(t, net.Config.Codec.UnmarshalJSON(out.Bytes(), &resp))
-				//require.NotNil(t, resp.Items)
+				// require.NotNil(t, resp.Items)
```
Comment (mustafapylons) [inline x/pylons/client/cli/query_list_item_by_owner_test.go:64]: Is the commented code for testing purpose or just code that needs to be removed?

File: x/pylons/client/cli/tx_execute_recipe_test.go [hunk 2]
```diff
@@ -675,32 +675,32 @@ func TestExecuteRecipeItemInputOutput(t *testing.T) {
...
+					Key:   "testInput",
+					Value: "testVal",
 				},
 			},
 		},
 	})
 	require.NoError(t, err)
 
-	//itemInputIDs, err := json.Marshal([]string{types.EncodeItemID(0)})
+	// itemInputIDs, err := json.Marshal([]string{types.EncodeItemID(0)})
```
Comment (mustafapylons) [inline x/pylons/client/cli/tx_execute_recipe_test.go:694]: same here, is it dead code or necessary for testing?

**Review by mustafapylons — COMMENTED** [review]
Few small comments, otherwise code is good.

**Review by BlackPanthar — COMMENTED** [review]
Reviewed. Ready to approve after Mustafa's comments are addressed.

File: x/pylons/client/cli/query_list_item_by_owner_test.go (hunk 1 above)
Comment (faddat) [reply x/pylons/client/cli/query_list_item_by_owner_test.go:64]: I haven't commented it.  the software has formatted the commented code.

File: x/pylons/client/cli/tx_execute_recipe_test.go (hunk 2 above)
Comment (faddat) [reply x/pylons/client/cli/tx_execute_recipe_test.go:694]: Same thing

**Comment by faddat** [comment]
hey, this was complete and comments have been addressed.

Please see new pr.

**Pull request closed without merging**